code/_compile_options.dm

```diff
@@ -14,14 +14,11 @@
 
 #ifdef LOCAL_GAME_TESTS
 #define GAME_TESTS
+#define MAP_TESTS
 #endif
 
-#ifdef CIBUILDING
-#define GAME_TESTS
-#endif
-
-#if defined(CIBUILDING) && defined(LOCAL_GAME_TESTS)
-#error CIBUILDING and LOCAL_GAME_TESTS should not be enabled at the same time!
+#if defined(GAME_TESTS) || defined(MAP_TESTS)
+#define TEST_RUNNER
 #endif
```

Comment:
This removes CI that checks that `LOCAL_GAME_TESTS` isnt on prod

Reply:
whoops, fixed